hackathon/urls.py

```diff
@@ -21,4 +21,3 @@
          HackathonDetailView.as_view(), name='hackathon_detail'),
     path('enroll', enroll_toggle, name='enroll_toggle')
 ]
```

Comment:
Was the blank line at the end of the file removed? It's generally required.

Reply:
Added the newline at the end.